mobile/android/android-emulator-services.ts

```diff
@@ -224,19 +226,26 @@ class AndroidEmulatorServices implements Mobile.IEmulatorPlatformServices {
 
 			// have to start new emulator
 			this.$logger.info("Starting Android emulator with image %s", image);
+			let emulatorProc: child_process.ChildProcess = null;
+			let emulatorProcEnded = false;
 			if(this.$options.geny) {
 				//player is part of Genymotion, it should be part of the PATH.
-				this.$childProcess.spawn("player", ["--vm-name", image],
-					{ stdio: "ignore", detached: true }).unref();
+				emulatorProc = this.$childProcess.spawn("player", ["--vm-name", image],
```

Comment:
this code is holding the terminal, in AppBuilder CLI we want to release the terminal when the emulator is successfully started, that's why we were using `.unref()`